src/assets/styles/variables.scss

```diff
@@ -6,6 +6,8 @@ $medium-gray: #99999a;
 $light-gray: #d4d4d4;
 $lighter-gray: #ededed;
 $lightest-gray: #f9f9f9;
+$text-gray: #656566;
+$navy-lighter:#A7CCED;
```

Comment:
> @ekarthiek For better readability, this would be a good opportunity to move all similar colors within proximity of eachother?
> 
> **Example:**
> 
> ```
> $gladiator: #c8102e;
> $gladiator-gradient: linear-gradient(215.85deg, #ea3855 9.33%, #c8102e 97.69%);
> 
> $pavement: #231f20;
> $dark-gray: #333;
> $text-gray: #656566;
> $medium-gray: #99999a;
> $light-gray: #d4d4d4;
> $lighter-gray: #ededed;
> $lightest-gray: #f9f9f9;
> 
> $navy: #263746;
> $navy-light: #3498dB;
> $navy-lighter:#A7cced;
> ```

@websmith I moved the variables with similar colors within proximity of each other